.changeset/button-layouts.md

```diff
@@ -0,0 +1,8 @@
+---
+"@jspsych/plugin-html-button-response": major
+"jspsych": patch
+---
+
+Button plugins now support either `display: grid` or `display: flex` on the container element that hold the buttons. If the layout is `grid`, the number of rows and/or columns can be specified. 
+
+jspsych.css has new layout classes to support this feature.
```

Comment:
Were you about to mention dropping `margin_horizontal` and `margin_vertical` here? :stuck_out_tongue:

Reply:
Oh right 🤦‍♂️